.golangci.yml

```diff
@@ -7,13 +7,23 @@ linters:
     - gosec
     - misspell
     - unparam
+    - interfacer
+    - nakedret
+    - gocyclo
+    - dupl
+    - goconst
+    - lll
 run:
   skip-dirs:
     # autogenerated clientset by client-gen
     - pkg/client
 linters-settings:
   errcheck:
     check-type-assertions: true
+  lll:
+    line-length: 250
+  dupl:
+    threshold: 400
```

Comment:
This seems like a very high threshold. The default is 15. I'm okay with having _some_ code duplication but 400 lines seems like too much. WDYT?

Reply:
yeah... I do agree.    This is what the controller/runtime project has... and tightening this results in work...